lib/beaker/hypervisor/ec2_helper.rb

```diff
@@ -19,6 +19,7 @@ def self.amiports(host)
 
       if roles.include? 'master'
         ports << 8140
+        ports << 8142
```

Comment:
Might also want 8143 for API access.

https://docs.puppet.com/pe/2017.2/sys_req_sysconfig.html#for-split-installs